BIPs/BIP-162/polygonTreasury.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": "1.0",
+  "chainId": "137",
+  "createdAt": 1674765035038,
+  "meta": {
+    "name": "Transactions Batch",
+    "description": "",
+    "txBuilderVersion": "1.11.1",
+    "createdFromSafeAddress": "0xeE071f4B516F69a1603dA393CdE8e76C40E5Be85",
+    "createdFromOwnerAddress": "",
+    "checksum": "0xec22caa1b2d9b174a176821b315b8b7dfc31ee33b0cd54a7eb48617741facbaf"
+  },
+  "transactions": [
+    {
+      "to": "0x0297e37f1873D2DAb4487Aa67cD56B58E2F27875",
```

Comment:
The DAO Treasury on Polygon at matic:0xeE071f4B516F69a1603dA393CdE8e76C40E5Be85 will initiate a transfer of “B-Polybase BPT” (contract 0x0297e37f1873D2DAb4487Aa67cD56B58E2F27875) in the amount of 26536547394777662718278 to the LM Multisig at matic:0xc38c5f97B34E175FFd35407fc91a937300E33860. From there, BPT will be withdrawn and all funds bridged back to Ethereum. USDC & WETH will be sent to the Karpatkey Safe 0x0EFcCBb9E2C09Ea29551879bd9Da32362b32fc89, BAL & WMATIC sent to the Treasury 0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f